deps/triemap/triemap.c

```diff
@@ -75,7 +74,7 @@ TrieMapNode *__trieMapNode_AddChild(TrieMapNode *n, char *str, tm_len_t offset,
   *__trieMapNode_childKey(n, n->numChildren - 1) = str[offset];
 
   __trieMapNode_children(n)[n->numChildren - 1] = child;
-  __trieNode_sortChildren(n);
+  n->flags &= ~TM_NODE_SORTED;
```

Comment:
@nermiller This change and other code changes are not expected.
I see lots of .c and .h files that are somehow included with this PR.

Reply:
Fixed!